### PR TITLE
EPMDJ-5389: added tests to the utility functions

### PR DESCRIPTION
**Pull request by IliaSolovev**
Issue: EPMDJ-5389

### CODE REVIEW

File: src/utils/percent-formatter.spec.ts [hunk 1]
```diff
@@ -9,6 +9,10 @@ describe('percentFormatter', () => {
     expect(percentFormatter(-213.20)).toEqual(-213.2);
   });
 
+  it('should return rounded to an integer number if the value is double number and after the decimal point equal to 0', () => {
```
Comment (pavyarov) [inline src/utils/percent-formatter.spec.ts:12]: there are `integer` and `double` at JS?)

Comment (pavyarov) [inline src/utils/percent-formatter.spec.ts:12]:
```suggestion
  it('should round to an integer number if the value is number with decimals and decimal part equal to 0', () => {
```

File: src/components/cells/duration-cell/duration.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export const duration = (value: number): { hours: string, minutes: string, seconds: string, affix: boolean } => {
```
Comment (pavyarov) [inline src/components/cells/duration-cell/duration.ts:1]: `duration`? maybe getDuration

Comment (pavyarov) [reply src/components/cells/duration-cell/duration.ts:1]: so now it's a separate function. so what is the `affix`?

Comment (pavyarov) [reply src/components/cells/duration-cell/duration.ts:1]: I mean, just imagine "tomorow" someone else will use this function. How he will know what `affix` means?

**Review by pavyarov — CHANGES_REQUESTED** [review]

File: src/components/cells/duration-cell/duration.ts (hunk 2 above)
Comment (IliaSolovev) [reply src/components/cells/duration-cell/duration.ts]: i renamed it like isLessThanOneSecond. Is it ok?

**Review by RomanAchilov — APPROVED** [review]

**Review by RomanAchilov — APPROVED** [review]

**Pull request merged**